Source/Plugins/Core/com.equella.core/js/tsrc/modules/CollectionsModule.ts

```diff
@@ -56,6 +55,7 @@ export const collectionListSummary = (
  * Find Collections by a list of ID.
  *
  * @param collectionUuids Collection UUIDs used to filter the list of all Collections.
+ * @returns { Collection[] | undefined } An array of Collections or undefined.
```

Comment:
This comment simply repeats the type definition, you want to explain what to expect and when.

```suggestion
 * @returns { Collection[] | undefined } An array of `Collection` instances matching the specified UUIDs or undefined if none could be found.
```